modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1153 @@
+
+.Example: using `+xss_sanitization+` to turn DOMPurify off and no longer sanitize HTML against XSS attacks
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  xss_sanitization: false
+});
+----
+
+[IMPORTANT]
+It is important to note that disabling sanitization introduces potential security issues, especially if the server-side XSS sanitization is weak or non-existent. Using this option therefore not recommended.
+
+Read more about xref:sanitizing-html-input-to-protect-against-xss-attacks[Sanitizing HTML input to protect against XSS attacks]
+
+=== Added top right close 'x' button of the modal dialogs to keyboard navigation
+//#TINY-9520
+
+In previous versions of {productname}, the top right close 'x' button within the modal dialogs were not keyboard-navigable by the Tab key.
+
+With the latest release of {productname} 6.4.1, users can now navigate through all dialogs that have a `x` close button. With these updates, users can now use the `Tab` key to move through the elements within the modal dialogue, making it more accessible and user-friendly. 
+
+This change is applied to all dialogs that have a `x` close button within {productname}.
+
+* Search and Replace
+* Accessibility Checker
+* Table Properties
+* Help
+* Source Code
+* URL
+* Image
+* Url Modal Dialog toolbar buttons. 
+
+[NOTE]
+Its worth nothing that the expected result for the Alert and Confirm dialogs is that the `initial focus` is set on the most appropriate button when the dialog is opened; for example, the “OK” button for Alert dialogs, and the “Yes” button for Confirm dialogs.
+
+In {productname} 6.4.1, these new additions/improvements are now available.
+
+=== New ui_mode option for editor in scrollable containers support
+//#TINY-9414
+
+A new `ui_mode: 'split'` option for scrollable containers has been introduced with the release of {productname} 6.4.1.
+
+This new option is designed to give users a better UX, by allowing them to navigate content within their editor easier with the use of the new ui_mode when integrated into their editors. This can be achieved by adding the `ui_mode: 'split'` option to the `tinymce.init()` within the `<script>` tags as below:
+
+.Basic integration for new ui_mode option using `split`
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  ui_mode: 'split',
+  min_height: 1400,
+});
+----
+
+[TIP]
+Its worth noting that when setting the new `ui_mode: 'split'` option for scrollable containers, when setting the height for the `<div class="scrollable-container">` ensure that the height is set to be less than `<` the `min_height` set of the {productname} editor container and by applying the `overflow: scroll;` CSS bevaviour to the `<div class="scrollable-container">`.
```

Comment:
```suggestion
Its worth noting that when setting the new `ui_mode: 'split'` option for editors within scrollable containers, such as within elements with the `overflow: scroll;` CSS property, ensure that the `min_height` of the editor is greater than the height of the outer container.
```
@shanmen-tiny What would this be for specifically, when the editor is in a scrolling container?

Reply:
@MitchCTiny @kemister85 I was talking to Karl before, and the content in the scrollable container needs to be greater than the height of the scrollable container so that the changes for `ui_mode: 'split'` is visible, and by setting the `min_height` of the editor.

Reply:
```suggestion
The new `ui_mode: 'split'` option alters the behaviour of editors within scrollable containers, where the `min_height` of the editor is greater than the height of the container element which has the `overflow="scroll"` style attribute.
```
Does this make more sense then?

Reply:
Im happy with that change. @shanmen-tiny happy with this?